services/cd-service/pkg/cmd/server.go

```diff
@@ -63,9 +66,51 @@ func (c *Config) readPgpKeyRing() (openpgp.KeyRing, error) {
 
 }
 
+// EnvConfig the server config that can be configured with env variables
+type EnvConfig struct {
+	//ProjectID       string `env:"GOOGLE_PROJECT_ID,required"`
+	EnvironmentName string `env:"ENV_NAME" envDefault:"local"`
+	TracingEnable   bool   `env:"TRACING_ENABLE" envDefault:"false"`
+	//ServiceAccountLoginPassword string `env:"SERVICE_ACCOUNT_LOGIN_PASSWORD,required"`
+	//ECDSAPublicKey              string `env:"JWT_PUBLIC_KEY,required"`
+	//CloudStorageBucketName      string `env:"CLOUD_STORAGE_BUCKET_NAME,required"`
+}
+
+// IsProduction true if this configuration is used in production, false otherwise
+func (s EnvConfig) IsProduction() bool {
+	return s.EnvironmentName == "production"
+}
+
+// IsStaging true if this configuration is used in staging, false otherwise
+func (s EnvConfig) IsStaging() bool {
+	return s.EnvironmentName == "staging"
+}
+
+// IsLocal true if this configuration is used locally, false otherwise
+func (s EnvConfig) IsLocal() bool {
+	return s.EnvironmentName == "local"
+}
```

Comment:
I don't see a use case for these functions and the names are actually flexible (it's not always "production"). -> remove